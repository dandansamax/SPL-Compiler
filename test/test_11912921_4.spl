char test_11912921_4(){
    int arrs[5;
    if(args);
    else;
    while(arrs6]==8){

    }
}

