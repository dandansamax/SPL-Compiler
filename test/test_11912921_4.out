Error type B at Line 2: Missing closing parenthesis ']'
Error type B at Line 5: Missing closing parenthesis ')'
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 3: Missing closing parenthesis '(' and ')'
Error type B at Line 3: Missing closing parenthesis '}'
